pkg/crc/machine/status.go

```diff
@@ -35,6 +35,10 @@ func (client *client) Status() (*types.ClusterStatusResult, error) {
 		return nil, errors.Wrap(err, "Cannot get machine state")
 	}
 
+	if vmStatus != state.Running {
+		return createClusterStatusResult(vmStatus, vm.bundle.GetBundleType(), vm.bundle.GetVersion(), "", 0, 0, 0, 0, 0, 0, nil)
```

Comment:
The code is quite cryptic, so an inline comment might be helpful to clearly indicate what happens. description is not enough top determine what the cluster result is.

Reply:
network-mode is not even an option on windows and mac, so probably we can merge https://github.com/crc-org/crc/pull/4514 and set linux as network-mode user too.

Then we can rework this to detect linux, and network-mode to system. That would clarify why quick early with a madeup cluster is needed

Reply:
add a comment in the code and also updated the commit description, as earlier it was wrong, the IP is actually needed to get the openshift cluster status

Reply:
how about following, this way it doesn't look cryptic and for both network mode it follows same path to get the IP since ram/disksize ..etc doesn't return the error but zero values.

```golang
[...]
	ip := ""
	if vmStatus == state.Running {
		ip, err = vm.IP()
		if err != nil {
			return nil, errors.Wrap(err, "Error getting ip")
		}
	}
[...]
```

Reply:
Thanks, updated the PR!